ads/OWNERS

```diff
@@ -4,7 +4,10 @@
 {
   rules: [
     {
-      owners: [{name: 'ampproject/wg-ads-reviewers'}],
+      owners: [
+        {name: 'ampproject/wg-ads-reviewers'},
+        {name: 'ampproject/wg-ads', requestReview: false},
```

Comment:
This won't quite work, as it will then not request reviews from anyone in the `wg-ads` user list. Instead, we have to do a little workaround here:
```javascript
ignoreThisIsForSyntaxHighlighting=
{
  rules: [
    {
      owners: [{name: 'ampproject/wg-ads'}]
    },
    {
      owners: [{name: 'ampproject/wg-ads-reviewers'}],
      pattern: '**/*'
    }
}
```
Patterned rules get higher priority, so this will try to pull from that pool first when assigning reviewers

Reply:
@zhouyx good catch.
@rcebulko thanks for the suggestion. i will make a separate PR with one file change to test the idea, unless you have a better testing method available

Reply:
Thanks @rcebulko for the clarification. Makes sense. Didn't know about the `pattern` feature. When there are multi patterns, is the more specific one going to be applied first?

Reply:
Priority: 
1. non-recursive file rule (ex. `pattern: 'package.json'`)
2. recursive file rule (ex. `pattern: '**/*.md'`)
3. directory rule (no pattern specified)

Reply:
Thanks！